arch/arm/soc/atmel_sam/same70/soc.c

```diff
@@ -227,6 +227,9 @@ static int atmel_same70_init(struct device *arg)
 
 	key = irq_lock();
 
+	SCB_EnableICache();
+	SCB_EnableDCache();
```

Comment:
I wonder if this should be:
```
        if (!(SCB->CCR & SCB_CCR_DC_Msk)) {
                SCB_EnableDCache();
        }
```
https://github.com/ARM-software/CMSIS_5/issues/331

Reply:
I agree that it should be done that way. Anyway given he amount of discussion in this PR, I wonder the commit enabling cache on SAM E70 should be move in a separate PR.